example/spec/parallel/dbRecords/DbRecordsDLQSpec.js

```diff
@@ -27,7 +36,7 @@ describe('when a bad record is ingested', () => {
   afterAll(async () => {
     await deleteS3Object(
       systemBucket,
-      failedMessageS3Key
+      leftoverS3Key
     );
   });
   it('is sent to the DLA and processed to have expected metadata fields', async () => {
```

Comment:
Can you break down this to multiple specs/it?

Reply:
I can, it duplicates a bunch of the work, but would make the test results clearer: doing it

Reply:
you can change `it` to `describe`, and define some variables under that describe to avoid more setups.

Reply:
done, thanks for the pointer